### PR TITLE
update digits_video.py

### PR DESCRIPTION
**Pull request by mabdullahrafique**
Following were the errors in the digits_video.py
     1 ) The code was not working because data type of x was float however in "cv2.rectangle" we require integer .
     2 ) After pressing the "esc" button the image windows did not destroy
So I amended following things:
     1 ) converted data type of x to int.
     2 ) used cv2.destroyAllWindows() to close all windows after the press of "esc" by user.

### CODE REVIEW

File: samples/python/digits_video.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ def main():
                 continue
             pad = max(h-w, 0)
             x, w = x-pad/2, w+pad
+            x= int(x)
```
Comment (alalek) [inline samples/python/digits_video.py]: Please use integer division instead on the line above:
```.diff
-pad/2
+pad//2
```

File: samples/python/digits_video.py [hunk 2]
```diff
@@ -98,3 +99,4 @@ def main():
 
 if __name__ == '__main__':
     main()
+cv2.destroyAllWindows()
```
Comment (alalek) [inline samples/python/digits_video.py]: This line should be places into `main()` function.

Looks like this observed effect is from IPython or similar. Normal standalone Python applications works well.

BTW, You could use `destroyWindow(name)` call instead:
https://stackoverflow.com/questions/15144442/when-is-an-opencv-imshow-window-destroyed-c

Comment (mabdullahrafique) [reply samples/python/digits_video.py]: Yes destroyAllWindows() can be called in main() function . Thanks

**Comment by alalek** [comment]
I pushed update for this PR.

**Pull request merged**